Apps/ValidationTests/Scripts/config.json

```diff
@@ -43,6 +43,7 @@
         {
             "title": "Black and White post-process",
             "playgroundId": "#N55Q2M#0",
+            "renderCount": 20,
```

Comment:
Is this an inherent part of the change, or was this motivated by the validation test strangeness we've been seeing recently?

Reply:
I did this to pass the test locally. It seems we need a couple of frames in both post-processing scenarios for the rendering to "settle." I have a feeling it's because we're calling `bgfx::touch` again-- we have the renderCount on chromatic aberration from when we had the touch call, but we didn't have it on this test which was added after the touch call was removed.